### PR TITLE
Update Ogden

### PR DESCRIPTION
**Pull request by zfergus**

### CODE REVIEW

File: src/polyfem/assembler/OgdenElasticity.hpp [hunk 1]
```diff
@@ -1,81 +1,82 @@
...
+	private:
+		template <typename T>
+		T elastic_energy_T(
+			const RowVectorNd &p,
+			const int el_id,
+			const DefGradMatrix<T> &def_grad) const;
 
-				val += 2 * mu / (alpha * alpha) * tmp;
-			}
+		Eigen::VectorXd alphas_;
```
Comment (teseoch) [inline src/polyfem/assembler/OgdenElasticity.hpp:43]: maybe this should the generic mat params

File: src/polyfem/assembler/OgdenElasticity.hpp [hunk 2]
```diff
@@ -1,81 +1,82 @@
...
+		Eigen::VectorXd mus_;
+		Eigen::VectorXd Ds_;
```
Comment (teseoch) [inline src/polyfem/assembler/OgdenElasticity.hpp:45]: and these too

File: src/polyfem/assembler/OgdenElasticity.hpp [hunk 3]
```diff
@@ -1,81 +1,82 @@
...
-		Eigen::VectorXd Ds_;
+		template <typename T>
+		T elastic_energy_T(
+			const RowVectorNd &p,
+			const int el_id,
+			const DefGradMatrix<T> &def_grad) const;
+
+		Eigen::VectorXd coefficients_;
+		Eigen::VectorXd expoenents_;
+		double bulk_modulus_;
```
Comment (teseoch) [inline src/polyfem/assembler/OgdenElasticity.hpp:80]: use generic elastic mat

File: tests/system_test_list.txt [hunk 4]
```diff
@@ -74,3 +74,8 @@ standard/plate_hole.json
 standard/navier_stokes_static.json
 standard/navier_stokes_transient.json
 standard/stokes_static.json
+standard/neohookean.json
+standard/mooney_rivlin.json
+#standard/mooney_rivlin_p2.json
```
Comment (zfergus) [inline tests/system_test_list.txt:79]: This one is not working with LDLT linear solver but works with the BiCGStab solver. I leave it as future debugging.

Comment (teseoch) [inline tests/system_test_list.txt:79]: why commented?

File: tests/test_assembler.cpp [hunk 5]
```diff
@@ -1,6 +1,5 @@
 #include <polyfem/State.hpp>
 
-#include <polyfem/assembler/GenericElastic.hpp>
 #include <polyfem/assembler/NeoHookeanElasticityAutodiff.hpp>
```
Comment (teseoch) [inline tests/test_assembler.cpp:4]: maybe add test of MR and O that matches NH?

File: tests/system_test_list.txt (hunk 4 above)
Comment (zfergus) [reply tests/system_test_list.txt:79]: This one is not working with LDLT linear solver but works with the BiCGStab solver. I leave it as future debugging.

**Review by teseoch — APPROVED** [review]

**Pull request merged**